docs/source/ko/add_new_model.md

```diff
@@ -148,21 +148,21 @@ source .env/bin/activate
 pip install -e ".[dev]"
 ```
 
-각 운영 체제에 따라 Transformers의 선택적 종속성 수가 증가하면이 명령으로 실패할 수 있습니다. 그런 경우에는 작업 중인 딥 러닝 프레임워크 (PyTorch, TensorFlow 및/또는 Flax)을 설치한 다음 다음 명령을 수행하면 됩니다:
+각 운영 체제에 따라 Transformers의 선택적 의존성이 개수가 증가하면 이 명령이 실패할 수 있습니다. 그런 경우에는 작업 중인 딥 러닝 프레임워크 (PyTorch, TensorFlow 및/또는 Flax)을 설치한 후, 다음 명령을 수행하면 됩니다:
```

Comment:
원본 : since the number of optional dependencies of Transformers is growing

AS-IS : 선택적 의존성이 개수가 증가하면
TO-BE : 선택적 의존성의 개수가 증가하면

조사를 바꾸는 방향은 어떨까요??

Reply:
오타낸 부분 짚어주셔서 감사합니다. 말씀하신 대로 수정하겠습니다!